src/resources/PanelLayout.py

```diff
@@ -0,0 +1,24 @@
+from rich.layout import Layout
+
+
+class PanelLayout:
+    """Define panel layout"""
+
+    def __init__(self) -> None:
+        pass
+
+    def make_layout() -> Layout:
```

Comment:
Since in the main code, you do ``PanelLayout.make_layout()``, you should decorate this with a classmethod, like this:
```suggestion
    @classmethod
    def make_layout() -> Layout:
```